customComponents/src/components/FooterLogo/styles.scss

```diff
@@ -1,3 +1,53 @@
+@import "../../../../../src/theme/styles/_variables.scss";
+@import "../../../../../src/theme/styles/base/structure/_index.scss";
```

Comment:
These import paths aren't ideal. Did you try just importing `theme/styles/_variables.scss`?